operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -3967,13 +4092,13 @@
           "targets": [
             {
               "editorMode": "builder",
-              "expr": "rate(watcher_workqueue_longest_running_processor_seconds_count{app=\"tekton-chains-controller\"}[5m])",
+              "expr": "rate(watcher_go_gc_cpu_fraction{app=\"tekton-chains-controller\"}[5m])",
```

Comment:
Not that I'm necessarily opposed to this change, but this was not one we discussed @jkhelil  ... explain to me why you went in this direction